asset/src/kafka_reader/slicer.ts

```diff
@@ -6,6 +6,10 @@ export default class KafkaSlicer extends Slicer<KafkaReaderConfig> {
         return Boolean(this.executionConfig.autorecover);
     }
 
+    isRestartable(): boolean {
```

Comment:
Are we certain that we are sticking with this name? I feel like once this name spreads out of Teraslice it will be harder to change.